scrapy/core/downloader/__init__.py

```diff
@@ -41,17 +41,17 @@ def close(self):
 
     def __repr__(self):
         cls_name = self.__class__.__name__
-        return "%s(concurrency=%r, delay=%0.2f, randomize_delay=%r)" % (
-            cls_name, self.concurrency, self.delay, self.randomize_delay)
+        return (f"{cls_name}(concurrency={self.concurrency!r}, "
+                f"delay={self.delay:.2f}, "
+                f"randomize_delay={self.randomize_delay!r}")
```

Comment:
@Gallaecio , I did not find where this class is tested. Would you please give me some directions to where/how would __repr__ be tested?

Reply:
I don’t think we have any. I would suggest creating a new file, e.g. `tests/test_core_downloader.py`, import `Slot` there, and add a simple test case class with a method that tests this.

Reply:
In 0e579182 I covered only the `__repr__` method, covering the whole file with tests would be great, but I think this would be outside the scope of this PR, won't you agree?